### DISCUSSION TITLE
Unlock security settings and unlock network settings with additional PIN (password)

### DISCUSSION THREAD

**Discussion by du8028934** [rustdesk/rustdesk#7656] [General]
I hope to add password verification when unlocking security settings and network settings. I'm not sure if I'm using super administrator privileges, so I don't need to verify. But in any case, changing these two settings requires verification and password verification.

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1770643072118095914

**Comment by HexHummingbird** [comment]
Password protected Network/Security settings would be great if you want to keep the app running but don't want a less technical person to see/change your relay server settings or permanent password, even if said person has admin Windows access. Great for kids and older family members' computers.

**Comment by rustdesk** [comment]
@21pages add this
- [ ] can set it on GUI and command line (both), this can also solve the problem of `pkexec` not working on some linux.
- [ ] encrypt it

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8977
